rskj-core/src/main/java/co/rsk/mine/MinerServerImpl.java

```diff
@@ -207,7 +210,7 @@ public void start() {
 
         synchronized (lock) {
             started = true;
-            blockListener = new NewBlockListener();
+            blockListener = new NewBlockTxListener(this.mainchainView, this, this.nodeBlockProcessor);
```

Comment:
Minor thing, why not removing `this.mainchainView` param? It can be retrieved from `this`, the second param.

Reply:
This would make to be necessary to write more code, getters in the interface, and the implementation or make the properties public.

Reply:
done